spec/round_spec.rb

```diff
@@ -0,0 +1,81 @@
+require './lib/helper.rb'
+
+RSpec.describe Round do
+  let!(:card1) {Card.new("What is the capital of Alaska?", "Juneau", :Geography)}
+  let!(:card2) {Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)}
+  let!(:card3) {Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)}
+  let!(:cards) {[card1, card2, card3]}
+  let!(:deck) {Deck.new(cards)}
+  let!(:round) {Round.new(deck)}
+
+  it 'is an instance of' do
+    expect(round).to be_instance_of(Round)
+  end
+
+  it 'has a deck' do
+    expect(round.deck.cards.count).to eq(3)
+  end
+
+  it 'can return cards from a category' do
+    expect(round.deck.cards_in_category(:STEM)).to eq([card2, card3])
+  end
+
+  it 'can check the current card' do
+    expect(round.current_card).to_not eq(deck.cards.last)
+  end
+
+  it 'can take a turn' do
```

Comment:
We should also test that the `take_turn` method returns a `Turn` object as per the spec. Right now it looks like this method returns an array so a test like this would have helped us catch that. 

```ruby  
it 'returns a turn object when take_turn' do
    expect(round.take_turn('Juneau')).to be_a(Turn)
  end
```